core/util.cjs

```diff
@@ -805,6 +819,7 @@ const UIStrings = {
   runtimeCustom: 'Custom throttling',
 };
 Util.UIStrings = UIStrings;
+Util.strings = {...UIStrings};
```

Comment:
In this case, it's because moving UIStrings declaration to be _before_ the `class` will be bad for `git blame` history. I'm not sure that's a particularly useful thing to call out in a comment tho